packages/validation/src/Questionary/boolean.ts

```diff
@@ -0,0 +1,13 @@
+import * as Yup from 'yup';
+
+export const booleanQuestionValidationSchema = (field: any) => {
```

Comment:
It's not good that we loose types here

Reply:
Yes but I am not sure how to keep them because this validation is separate repository and I don't know if there is any way to keep the types in both or inject them somehow.